modules/openapi-generator-gradle-plugin/src/main/kotlin/org/openapitools/generator/gradle/plugin/tasks/GenerateTask.kt

```diff
@@ -98,7 +99,8 @@ open class GenerateTask : DefaultTask() {
      * The template directory holding a custom template.
      */
     @Optional
-    @Input
+    @InputDirectory
+    @PathSensitive(PathSensitivity.RELATIVE)
```

Comment:
You might also consider `@IgnoreEmptyDirectories` and `@NormalizeLineEndings` if those things can be ignored in these inputs, too.

Reply:
It looks `@IgnoreEmptyDirectories` and `@NormalizeLineEndings` are only available in Gradle 6.8 & Gradle 7.2, respectively. This project is building with Gradle 5.6.4.

Reply:
Ah, yes, backwards compatibility spoils the day, once again...

Reply:
dang, if it were built with a newer version you could still add the annotations and get all the backwards compatibility since annotations won't break backwards compatibility.

Reply:
I found #12719 which intends to upgrade the project to Gradle 7.4.2, though it appears stale. 😞